packages/components/tests/dummy/app/templates/components/form/sensitive-input.hbs

```diff
@@ -0,0 +1,303 @@
+    <SG.Item @label="Label + Required">
+      <Hds::Form::SensitiveInput::Field @isMultiline={{true}} @value="Lorem ipsum dolor" @isRequired={{true}} as |F|>
+        <F.Label>This is the label text</F.Label>
+      </Hds::Form::SensitiveInput::Field>
+    </SG.Item>
+    <SG.Item @label="Label + Optional">
+      <br />
+      <Hds::Form::SensitiveInput::Field @isMultiline={{true}} @value="Lorem ipsum dolor" @isOptional={{true}} as |F|>
+        <F.Label>This is the label text</F.Label>
+      </Hds::Form::SensitiveInput::Field>
+    </SG.Item>
+  </Shw::Grid>
+
+  <Shw::Divider @level="2" />
+
+  <Shw::Text::H3>Containers</Shw::Text::H3>
+
+  <Shw::Text::H4>Single line</Shw::Text::H4>
+
+  <Shw::Grid @columns={{3}} as |SG|>
+    {{#let (array "block" "flex" "grid") as |displays|}}
+      {{#each displays as |display|}}
+        <SG.Item>
+          <Shw::Label>Parent with <code>display: {{display}}</code></Shw::Label>
+          <Shw::Flex as |SF|>
+            <SF.Item @grow={{true}} {{style display=display}}>
+              <Hds::Form::SensitiveInput::Field @value="Default width" as |F|>
+                <F.Label>This is the label</F.Label>
+                <F.HelperText>This is the helper text</F.HelperText>
+              </Hds::Form::SensitiveInput::Field>
+            </SF.Item>
+            <SF.Item @grow={{true}} {{style display=display}}>
+              <Hds::Form::SensitiveInput::Field @value="Custom width" @width="120px" @isInvalid={{true}} as |F|>
+                <F.Label>This is the label text that should go on multiple lines</F.Label>
+                <F.HelperText>This is the helper text that should go on multiple lines</F.HelperText>
+                <F.Error as |E|>
+                  <E.Message>This is the first error text</E.Message>
+                  <E.Message>This is the second error text that should go on multiple lines</E.Message>
+                </F.Error>
+              </Hds::Form::SensitiveInput::Field>
+            </SF.Item>
+          </Shw::Flex>
+        </SG.Item>
+      {{/each}}
+    {{/let}}
+  </Shw::Grid>
+
+  <Shw::Text::H4>Multiline</Shw::Text::H4>
+
+  <Shw::Grid @columns={{3}} as |SG|>
+    {{#let (array "block" "flex" "grid") as |displays|}}
+      {{#each displays as |display|}}
+        <SG.Item>
+          <Shw::Label>Parent with <code>display: {{display}}</code></Shw::Label>
+          <Shw::Flex as |SF|>
+            <SF.Item @grow={{true}} {{style display=display}}>
+              <Hds::Form::SensitiveInput::Field @isMultiline={{true}} @value="Default width" as |F|>
+                <F.Label>This is the label</F.Label>
+                <F.HelperText>This is the helper text</F.HelperText>
+              </Hds::Form::SensitiveInput::Field>
+            </SF.Item>
+            <SF.Item @grow={{true}} {{style display=display}}>
+              <Hds::Form::SensitiveInput::Field
+                @isMultiline={{true}}
+                @value="Custom width"
+                @width="120px"
```

Comment:
One thing I noticed, is that in the `Textarea` we don't test the `@height` argument. Maybe we could do it this context (here and also in the `Textarea`)?

Reply:
Added in [6361cd8](https://github.com/hashicorp/design-system/pull/1468/commits/6361cd83e7bf6fd349a1df8a1fbce220319a109a)